google/calendar/credential/tool.gpt

```diff
@@ -0,0 +1,21 @@
+Name: Google OAuth Credential
+Share Credential: google-cred as google-calendar
+Type: credential
+
+---
+Name: google-cred
+Tools: ../../../oauth2
+
+#!sys.call ../../../oauth2
+
+{
+"oauthInfo": {
+	"integration": "google-calendar",
+	"token": "GOOGLE_OAUTH_TOKEN",
+	"scope": [
+		"https://www.googleapis.com/auth/userinfo.email",
```

Comment:
@tybalex tbh I haven't tested the scopes. I think you will need to double check on this

Reply:
ok

Reply:
yeah it is not necessary, removed